frontend/src/pages/ServiceDetails/ServiceDetailsPage.tsx

```diff
@@ -49,7 +49,7 @@ type ServiceDetailsState = {
 interface ServiceDetailsProps {
   serviceId: ServiceId;
   duration: DurationInSeconds;
-  jaegerInfo?: JaegerInfo;
+  jaegerInfo?: TracingInfo;
```

Comment:
If the type is TracingInfo, maybe the variable should be renamed to tracingInfo.